plugins/auth-backend-module-oidc-provider/src/authenticator.ts

```diff
@@ -85,6 +88,16 @@ export const oidcAuthenticator = createOAuthAuthenticator({
       );
     }
 
+    const timeoutHumanDuration = config.getOptional<HumanDuration>('timeout');
```

Comment:
Can we update this to https://github.com/backstage/backstage/blob/master/packages/config/src/readDurationFromConfig.ts#L47?

Reply:
like this 
```typescript
const timeoutMilliseconds = config.has('timeout')
      ? durationToMilliseconds(
          readDurationFromConfig(config, { key: 'timeout' }),
        )
      : undefined;
```

Reply:
@deejay1 @aramissennyeydd thank you for your comments! Just updated the code. Please have a look. Have a nice one 🙂

Reply:
@goququ please check the errors in the test